pkg/util/util.go

```diff
@@ -108,11 +109,17 @@ func Forever(f func(), period time.Duration) {
 	Until(f, period, NeverStop)
 }
 
-// Until loops until stop channel is closed, running f every period.
+// Until is syntactic sugar on top of JitterUntil with zero jitter factor
+func Until(f func(), period time.Duration, stopCh <-chan struct{}) {
+	JitterUntil(f, period, 0.0, stopCh)
```

Comment:
so unless someone starts using JitterUntil, that part "if jitterFactor > 0.0" will be deadcode for time being, it seems.